site/sfguides/src/vhol_iceberg_snowflake_open_catalog/vhol_iceberg_snowflake_open_catalog.md

```diff
@@ -0,0 +1,575 @@
+- VS Code (optional, for Spark integration)
+
+<!-- ------------------------ -->
+## Setting Up Snowflake Open Catalog
+
+- Sign in to your Snowflake account
+- Change your active role to ORGADMIN
+- Click Admin -> Accounts
+- Click the dropdown arrow next to “+ Account” in the top right
+
+![create snowflake open catalog account](assets/img01.PNG)
+
+- In the dialog box that pops up:
+    For “Cloud”, choose “AWS”
+    For “Region”, choose “US West-2”
+    Don’t change the “Edition”, leave it as default
+    Click “Next”
+- In the next screen:
+    For “Account Name”, type in “DECONNECT_HOL_DEMO” 
+    For “User Name”, type in “admin”
+    For “Password”, type in a password that meets the requirements (Must be at least 8 characters long, contain at least 1 digit, 1 uppercase letter and 1 lowercase letter). Note you’ll have to change this password on first login
+    For “Email”, type in your Snowflake email address
+    Click “Create Account”
+- In the next step, you should see “Account created successfully”
+    Copy the “Account Locator URL” value and paste it in the HOL spreadsheet
+    Now click on the value for “Account Locator URL” and it’ll open in a new tab. Note you may get a 404 error while the account is being provisioned. Wait a minute or two and refresh the page and it’ll go away
+    Enter your user name and password and login
+    If you have MFA set up on your Snowflake account, you’ll be prompted to set up MFA for this account. Go through the steps to set that up.
+    You’ll be prompted to reset your password. Enter a new password that meets the same requirements (and can’t be the same one you set in step 1.f.iii)
+    Once you click “Reset”, you’ll be logged in and redirected to the home page of Snowflake Open Catalog
+
+![open catalog home page](assets/img02.PNG)
+
+<!-- ------------------------ -->
+
+## Creating an S3 Bucket
+
+- Open a new tab and sign in to the AWS Management Console.
+- In the top right corner, change your region to “US West-2”
+- On the home dashboard, search for and select S3.
+- Select Create bucket.
+- On the Create bucket page:
+    For Bucket name, enter “deconnect-hol-demo-<name>” where <name> is your name (this is just to make it unique since you bucket names have to be unique globally, feel free to put whatever you want as a suffix, just only use characters, numbers, and hyphens)
+    Leave the rest of the configuration default
+    Select Create bucket.
+- Once it’s created successfully, on the next screen (S3 buckets list), click the name of the S3 bucket you created
+- On the right, click “Create folder”
+- On the next screen:
+    For “Folder name”, enter “snowflake-managed-iceberg”
+    Leave the rest of the configurations as default
+    Click “Create folder”
+- Click the checkbox next to the folder name you just created. Then click “Copy S3 URI”. Paste this value in the HOL spreadsheet.
+
+<!-- ------------------------ -->
+
+## Create an IAM policy that grants access to your S3 location
```

Comment:
@vinodhini-sd Shorten and title case the step label so the left-nav looks cleaner